README.md

```diff
@@ -14,30 +14,49 @@ This repo represents the Community Automation effort where teams can contribute
 
 A public docker image has been create with all prereq's for running playbooks.
 
-**MAC/RHEL7** users need to install docker to run this option
+Make sure docker or podman(RHEL 8) is installed and running on your workstation
 
 **NOTES:**
 
-- On RHEL8 use podman
 - You may need to do a **docker logout** before you begin.
 - default ssh keys are comming from your ~/.ssh/id_rsa and ~/.ssh/id_rsa.pub
 
 **Tested on MAC (big sur), Ubunutu 16.04,18.04,20.04, and RHEL 7/8**
 
+## MAC, Ubuntu, and RHEL 7 users
+
+```
+# docker run -v /YOUR_REPO_PATH:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest
+```
+
+### calling with playbook, assumes all vars are set in vars files
+
+```
+# docker run -v /YOUR_REPO_PATH:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest -c "ansible-playbook -i YOUR-PLAY/inventory  YOUR_PLAYBOOK/playbook.yml"
+```
+
+### using param passing, a mix of vars files, and command line vars
+
+```
+# docker run -v /YOUR_REPO_PATH:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest -c "ansible-playbook -i YOUR-PLAY/inventory  YOUR_PLAYBOOK/playbook.yml -e \"var1=value1\" -e \"var2=value2\""
 ```
-# RHEL 7: sudo systemctl start docker; sudo yum install -y docker
-# docker run -v /YOUR_REPO_PATH:/community-automation -i -t quay.io/rayashworth/community-ansible:latest
 
-NOTE: RHEL 8
-# podman run -v /YOUR_REPO_PATH:/community-automation -i -t quay.io/rayashworth/community-ansible:latest
+## RHEL 8
+
+```
+# podman run -v /YOUR_REPO_PATH:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest
```

Comment:
Might want to state this is to be used to create a  local environment to run in

podman run -v <YOUR_REPO_ABSOLUTE_PATH>:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest

Reply:
done